style.css

```diff
@@ -1,7 +1,13 @@
+:root{
+  --main-bg-color: #151F2B;
+  --border-color: #38444D;
+  --items-bg-color: #192734;
+  --text-color: #ffffff;
```

Comment:
You didn't use this text color, and you already have some text in place in the content which is still  black